### PR TITLE
Chromium Chronicle 21

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- Adds Chromium Chronicle 21

### CODE REVIEW

**Comment by petele** [comment]
CC: @mkamesam3

**Comment by kyleshima** [comment]
Not too familiar with reviewing stuff on github, but noticed one small thing. On L52 can you change
`func(context.Context)` error
to
`func(context.Context) error`

The code block should include "error". Everything else LGTM

**Comment by petele** [comment]
@kyleshima - thanks for pointing that out, I've fixed.

File: site/en/blog/chromium-chronicle-21/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+title: "The Chromium Chronicle #21: Chrome OS E2E UI Automation"
+description: >
+  Tast is a new UI library has been created that uses the Chrome a11y tree
```
Comment (atetubou) [inline site/en/blog/chromium-chronicle-21/index.md:4]: Can we leave non-abbreviation form of a11y in its first appearence?

```suggestion
  Tast is a new UI library has been created that uses the Chrome a11y (accessibility) tree
```

File: site/en/blog/chromium-chronicle-21/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+tags:
+  - chromium-chronicle
+---
+
+**Episode 21:** by Brendan Hansknecht in Mountain View, CA (May, 2021)<br>
+[Previous episodes](/tags/chromium-chronicle/)
+
+Historically, automating the Chrome OS UI in E2E tests across the entire
+fleet of devices has been difficult. In Tast, a new UI library has been
+created that uses the Chrome a11y tree to control the Chrome OS UI. **This
```
Comment (atetubou) [inline site/en/blog/chromium-chronicle-21/index.md:21]:
```suggestion
created that uses the Chrome a11y (accessibility) tree to control the Chrome OS UI. **This
```

File: site/en/blog/chromium-chronicle-21/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+---
+title: "The Chromium Chronicle #21: Chrome OS E2E UI Automation"
```
Comment (mkamesam3) [inline site/en/blog/chromium-chronicle-21/index.md:2]: Got some feedback from a reviewer that "E2E" might be ambiguous to some readers. Could we please change this instance in the title to "End-to-end"?

File: site/en/blog/chromium-chronicle-21/index.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+description: >
+  Tast is a new UI library has been created that uses the Chrome a11y tree
+  to control the Chrome OS UI. The library enables developers to easily
+  create E2E tests against any visible UI surface.
```
Comment (mkamesam3) [inline site/en/blog/chromium-chronicle-21/index.md:6]: nit: Could this be changed to "End-to-end (E2E)"? This way, maybe other instances of E2E used later on can remain as they are, after the acronym is established.

File: site/en/blog/chromium-chronicle-21/index.md (hunk 3 above)
Comment (petele) [reply site/en/blog/chromium-chronicle-21/index.md]: Done!

**Review by robdodson — APPROVED** [review]

**Pull request merged**